The codemadness frontends interface has changed as of 0.8 (current
release in postmarketOS stable and alpine). youtube-cli now has different stdout.

~ $ youtube-cli  "big buck bunny"
           Big Buck Bunny 60fps 4K - Official Blender Foundation Short Film [10:35]
URL:       https://www.youtube.com/embed/aqz-KE-bpKQ
Channel:   Blender: https://www.youtube.com/feeds/videos.xml?channel_id=UCSMOQeBJ2RAnuFungnQOxLg
Published: 10 years ago
Views:     21,991,421

           Big Buck Bunny [9:57]
URL:       https://www.youtube.com/embed/YE7VzlLtp-4
Channel:   Blender: https://www.youtube.com/feeds/videos.xml?channel_id=UCSMOQeBJ2RAnuFungnQOxLg
Published: 17 years ago
Views:     8,798,926

           Big Buck Bunny test [1:20]
URL:       https://www.youtube.com/embed/uPN7PrswisM
Channel:   VideoLink: https://www.youtube.com/feeds/videos.xml?channel_id=UCF1hSdl8-2Zr4AJf9AZOOAQ
Published: 11 years ago
Views:     95,012

This patch fixes the script for the new output
---
 scripts/appscripts/sxmo_youtube.sh | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/scripts/appscripts/sxmo_youtube.sh b/scripts/appscripts/sxmo_youtube.sh
index 0a6081e..51979b3 100755
--- a/scripts/appscripts/sxmo_youtube.sh
+++ b/scripts/appscripts/sxmo_youtube.sh
@@ -12,11 +12,10 @@ AUDIOONLY=0
 youtubesearch() {
 	QUERY="$1"
 	youtube-cli "$QUERY" |
-		grep -Ev '^(Channelid|Atom feed|Channel title|Published|Viewcount|Userid):' |
-		sed -E 's/^(URL|Duration):\s+/\t/g' |
-		tr -d '\n' |
-		sed 's/===/\n/g' |
-		gawk -F'\t' '{ print $3 " " $1 " " $2}'
+		grep -Ev '^(Channel|Published|Views):' |
+		sed -E 's/^URL:\s+/\t/g' |
+		awk '{$1=$1};1' |
+		paste - - -
 }
 
 searchmenu() {
-- 
2.49.0


